templates/index.html

```diff
@@ -42,6 +42,7 @@
 
   {# ITALIAN #}
   {% include "takeovers/it/_da-vmware-a-openstack.html" %}
+  {% include "takeovers/_da-vmware-a-openstack.html" %}
```

Comment:
I know I said it would be fine, but looking at it I think this takeover should be moved to the /it directory and renamed to something like `_da-vmware-a-charmed-openstack.html`